deployment/GP03FLMB_Deploy.csv

```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMB-RIS01-00-SIOENG000,6,1,2018-07-24T23:01:00,2019-09-
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIS01-04-PHSENF000,7,1,2019-09-27T21:26:00,2021-07-27T18:00:00,CGMGP-03FLMB-00007,,CGINS-PHSENF-P0147,50.37555,-144.5145,,30,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIS01-05-FLORTD000,7,1,2019-09-27T21:26:00,2021-07-27T18:00:00,CGMGP-03FLMB-00007,,CGINS-FLORTD-01298,50.37555,-144.5145,,30,4176,,,
+SKQ202111S,,,,GP03FLMB-RIM01-00-SIOENG000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-13283,50.331522,-144.480312,,1527,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-ADCPSL007,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-ADCPSL-20502,50.331522,-144.480312,,520,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG060,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13624,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG061,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13421,50.331522,-144.480312,,60,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG062,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13427,50.331522,-144.480312,,80,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG063,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13426,50.331522,-144.480312,,110,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG064,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13425,50.331522,-144.480312,,150,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG065,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13422,50.331522,-144.480312,,200,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG066,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-12190,50.331522,-144.480312,,270,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG067,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13431,50.331522,-144.480312,,370,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG068,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13429,50.331522,-144.480312,,520,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH069,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13631,50.331522,-144.480312,,770,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH070,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-12226,50.331522,-144.480312,,1020,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH071,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13632,50.331522,-144.480312,,1520,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-14883,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0180,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-05-FLORTD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-FLORTD-01324,50.331522,-144.480312,,50,4166,,,
```

Comment:
Deployment times and locations look good.  Looks like deployment depths were increased by 20 m due to the mooring being deployed 21 m deeper than designed.  We don't usually tweak the design depths, but given the magnitude of the different, this looks good.  I wonder if we should add a note to those that they were "Deployed 20 m deeper than the design depth."